tree/10_ncf_internals/configuration.cf

```diff
@@ -79,6 +79,18 @@ bundle agent configuration
       "pass2" expression => "pass1";
       "pass1" expression => "any";
 
+      # Make log level incremental
+      "debug"        expression => "trace",
+                          scope => "namespace";
+      # CFEngine's stdlib verbose mode
+      "verbose_mode" expression => "debug",
+                          scope => "namespace";
+      "info"         expression => "debug",
```

Comment:
In ncf, we have:
* *info*
* *debug*
* *trace*

CFEngine has:
* *inform_mode*
* *verbose_mode*
* *debug_mode*

We should at least define ncf classes when CFEngine classes are defined.